pkg/security/ebpf/c/exec.h

```diff
@@ -507,7 +512,7 @@ int sched_process_fork(struct _tracepoint_sched_process_fork *args) {
     }
 
     u64 ts = bpf_ktime_get_ns();
-    struct exec_event_t *event = get_exec_event();
+    struct exec_event_t *event = get_zero_exec_event();
     if (event == NULL) {
```

Comment:
Maybe we can remove those checks now ?

Reply:
I don't think we can remove, this can be still NULL

Reply:
Oh yeah you're right !